feature1/one-more-file.txt

```diff
@@ -0,0 +1,4 @@
+This file was also added in the feature 1 branch.
```

Comment:
It is really good you decided to add this line

Reply:
Thanks. Rust.